cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -146,6 +148,62 @@ type Member struct {
 	PeerURL string
 }
 
+// GetMemberID returns the member ID of the given peer URL
+func (c Client) GetMemberID(peerURL string) (uint64, error) {
```

Comment:
The other func's are declared with (c *Client)
Should these two follow the same pattern ?

Reply:
@tedyu Thanks for you point out it. 
Now there are different signatures in etcd.go. This PR implement remove local etc member from cluster, so keep function have the same signatures is not in the scope of this PR. I will open an issue to track it. You can attention it.